x-pack/legacy/plugins/ml/public/management/jobs_list/index.ts

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+// @ts-ignore no declaration module
+import { ReactDOM, render, unmountComponentAtNode } from 'react-dom';
+import routes from 'ui/routes';
+import { canGetManagementMlJobs } from '../../privilege/check_privilege';
+import { JOBS_LIST_PATH } from '../management_urls';
```

Comment:
Looks like some imports could be grouped here so there's only one statement for each referenced file?